src/navigation/wallet/screens/WalletDetails.tsx

```diff
@@ -1195,15 +1193,9 @@ const WalletDetails: React.FC<WalletDetailsScreenProps> = ({route}) => {
         ItemSeparatorComponent={() => <BorderBottom />}
         ListFooterComponent={listFooterComponent}
         onEndReached={() => {
-          if (!onEndReachedCalledDuringLoading) {
-            loadHistory();
-            setOnEndReachedCalledDuringLoading(true);
-          }
-        }}
-        onEndReachedThreshold={0.5}
```

Comment:
was it intentional to change threshold from 0.5 to .01?

Reply:
Thanks for review! It wasn't intentional. I was testing different values and forgot to keep in 0.5 in order to load at the half of elements.